modules/paragraphs_riddle_marketplace/src/Controller/RiddleUrlAutocompleteController.php

```diff
@@ -80,7 +80,7 @@ public function autocomplete(Request $request) {
    *
    * @param string $typedRiddleTitle
    *   Search text to match Riddle Title.
-   * @param array $riddleFeed
+   * @param array|null $riddleFeed
```

Comment:
Why not make the RiddleFeedService::getFeed() method always return an (possibly empty) array instead of handling the possibility of having either array or null in all those methods? I think that would be nicer.